index.js

```diff
@@ -30,6 +40,49 @@ app.get('/ideas', (req, res) => {
     res.status(200).json(getIdeas());
 });
 
+app.get('/ideas/sorted/donationSum', (req, res) => {
+    res.status(200).json(sortedByDonationSumIdeas());
+});
+
+app.get('/ideas/sorted/pendingStatus', (req, res) => {
+    res.status(200).json(pendingIdeasList());
+});
+
+app.get('/ideas/sorted/approvedStatus', (req, res) => {
+    res.status(200).json(approvedIdeasList());
+});
+
+app.post('/ideas', (req, res) => {
+    res.status(200)
+        .json(
+            createIdeas(
+                req.body.header,
+                req.body.description,
+                req.body.askedSum,
+                req.body.userId
+            )
+        )
+        .send('Idea created successfully.');
+});
+
+app.put('/ideas', (req, res) => {
+    res.status(200)
+        .json(updateIdea(req.body.ideaId, req.body.key, req.body.value))
```

Comment:
Can `updateIdea` function throw an error?

Reply:
Updated and tested.